packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -482,6 +491,19 @@ export function ArticleEditor({id}: ArticleEditorProps) {
                   )}
                 </div>
               }
+              rightChildren={
+                <IconButtonLink
+                  disabled={hasChanged || !id}
+                  style={{marginTop: '4px'}}
+                  size={'lg'}
+                  icon={<Icon icon="eye" />}
+                  onClick={async e => {
+                    await previewLinkRefetch({id: id, hours: 1})
```

Comment:
`previewLinkRefetch()` returns the data here and you should use that link to open a new window or not.